### DISCUSSION TITLE
Allow customizing title of right sidebar

### DISCUSSION THREAD

**Discussion by aantn** [pradyunsg/furo#288] [Ideas] [answered]
I'd like to change the title of the right sidebar from "Contents" to "On this page". I find it creates a more immediate and intuitive understanding of why there are two sidebars and how they differ.

**Comment by aantn** [comment]
Some additional references:

* [This Furo field report mentions users confused by the sidebar titles](https://github.com/pradyunsg/furo/issues/98)
* [The Vector.dev docs use "On this page"](https://vector.dev/docs/)

**Comment by pradyunsg** [comment, marked as answer]
With the power of "I have more experience now, and there's a better choice here": https://github.com/pradyunsg/furo/commit/7933eb886b13890342cfa7e9a0030bd8f034fdf0

As for making this configurable... I definitely don't wanna add knobs to Furo for every little thing -- each knob provided to the user is additional maintainance overhead and complexity in the implementation, which make continued maintainance and development more difficult. This is relatively straightforward and cheap to change, and there's a clearly-better choice for this. :)

**Reply by aantn** [reply]
Thanks, "On this page" is perfect.
